docs/output.md

```diff
@@ -27,6 +27,7 @@ The pipeline is built using [Nextflow](https://www.nextflow.io/) and processes d
                 * [Alpha diversity indices](#alpha-diversity-indices) - Diversity within samples
                 * [Beta diversity indices](#beta-diversity-indices) - Diversity between samples (e.g. PCoA plots)
             * [ANCOM](#ancom) - Differential abundance analysis
+        * [PICRUSt2](#picrust2) - Predicting functional abundances
```

Comment:
Or "Predicting functional genes", "Predicting functional gene abundances"?

Reply:
I changed to "Predict the functional potential of a bacterial community"